apps/judicial-system/web/src/routes/Court/RestrictionCase/Confirmation/Confirmation.tsx

```diff
@@ -75,6 +77,19 @@ export const Confirmation: React.FC = () => {
     }
   }
 
+  const handleNextButtonClick = async () => {
+    const isCorrectingRuling = workingCase.notifications?.some(
```

Comment:
We treat the case as reopened only if the parties to a case have been notified of a signed ruling. This happens after the ruling is signed by a judge. As a consequence we do not treat a reopened case which ruling has not been signed as being reopened; it is treated as if it was just received.